references/recaptcha-enterprise/sf-recaptcha-enterprise-v1/sharedflowbundle/policies/EV-ExtractReCaptchaToken.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2022 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<ExtractVariables name="EV-ExtractReCaptchaToken">
```

Comment:
Nice to have but I don't see a difference to using the `request.header.x-recaptcha-token` variable directly.

I guess that's a style debate and performance should be negligible in this case.

Reply:
Yes, it is more for the consistency of the configuration / flow variables